site/content/docs/main/restore-hooks.md

```diff
@@ -0,0 +1,157 @@
+---
+title: "Restore Hooks"
+layout: docs
+---
+
+Velero now supports Restore Hooks, custom actions that can be executed during or after the restore process. There are two kinds of Restore Hooks:
+
+1. InitContainerRestoreHooks: These will add init containers into restored pods to perform any necessary setup before the application containers of the restored pod can start.
+1. ExecRestoreHooks: These can be used to execute custom commands or scripts in containers of a restored Kubernetes pod.
+
+Only InitContainerRestoreHooks is supported and support for ExectRestoreHooks is COMING SOON!
+
+## InitContainer Restore Hooks
+
+As previously mentioned, these hooks can be used to add init containers into pods, before them being restored, to have the init containers perform preparatory actions for the restored pod to resume running from its backed-up state.
+
+There are two ways to specify `InitContainer` restore hooks:
+1. Specifying restore hooks in annotations
+1. Specifying restore hooks in the restore spec
+
+### Specifying Restore Hooks As Pod Annotations
+
+Below are the annotations that can be added to a pod to specify restore hooks:
+* `init.hook.restore.velero.io/container-image`
+    * The container image for the init container to be added.
+* `init.hook.restore.velero.io/container-name`
+    * The name for the init container that is being added.
+* `init.hook.restore.velero.io/command`
+    * This is the `ENTRYPOINT` for the init container being added. This command is not executed within a shell and the container image's `ENTRYPOINT` is used if this is not provided.
+
+#### Example
+
+Use the below commands to add annotations to the pods before taking a backup.
+
+```bash
+$ kubectl annotate pod -n <POD_NAMESPACE> <POD_NAME> \
+    init.hook.restore.velero.io/container-name=restore-hook \
+    init.hook.restore.velero.io/container-image=alpine:latest \
+    init.hook.restore.velero.io/command='["/bin/ash", "-c", "date"]'
+```
+
+The above annotation will result in the below init container to be added to the pod when it is restored.
```

Comment:
"With the annotation above, Velero will add the following init container to thee pod when its restored"

Rewrite suggestion to use active voice